setup.cfg

```diff
@@ -17,3 +17,22 @@
 
 [flake8]
 max-line-length=99
+
+[metadata]
+name = apache-airflow-client
+description = Apache Airflow API (Stable)
+author = Apache Software Foundation
+author_email = dev@airflow.apache.org
+url = https://airflow.apache.org/
+long_description = file: README.md
+long_description_content_type = text/markdown
+license = Apache License 2.0
+project_urls =
+    Documentation=https://airflow.apache.org/docs/
```

Comment:
```suggestion
    Documentation=https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html
```

Reply:
I feel we should have separate docs for the python-client so that we can also have an Updating guide

Reply:
coz https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html has the version of the API and it might be confusing

Reply:
The link is with "stable" and I think it is good as is - API is more of a property of Airflow not the client, so it conceptually belongs to Airflow.

Reply:
But yeah, I agree at some point in time we might want to have "Updating guide" for the client. So I agree eventually we should have a section for clients like we have "providers" and "docker-stack" now.

Reply:
In this case we should likely  have a completely separate "Airflow API" section and have sub-sections for:

```
1) API specs
2) Clients
    - Python 
    - Go ....
```

Reply:
Exactly my point :)